buildconfig/CMake/CppCheck_Suppressions.txt.in

```diff
@@ -733,7 +733,7 @@ constVariableReference:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/SaveFullpr
 passedByValue:${CMAKE_SOURCE_DIR}/Framework/Crystal/src/SaveHKL.cpp:579
 passedByValue:${CMAKE_SOURCE_DIR}/Framework/Crystal/src/SaveHKL.cpp:580
 constVariableReference:${CMAKE_SOURCE_DIR}/Framework/Crystal/src/SaveHKL.cpp:158
-constVariableReference:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadNexusProcessed.cpp:922
+constVariableReference:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadNexusProcessed.cpp:923
```

Comment:
Please fix the suppression by adding the word `const` to the right line and deleting this line.

Reply:
I'll add the `const`.  **But, my problem with this was that that line has to do with a _modification_, so it's definitely _not_ `const` -- that's why I preferred the suppression!**

Reply:
My misunderstanding. Please move the supression to be inline cpp file. Things that are going to stay should get moved over. Things in the suppressions file are meant to be dealt with when people want to go weeding.

Reply:
Added the suppression `inline`.  I also checked the signature:  definitely should _not_ be `const`.